tests/test_normalization.py

```diff
@@ -0,0 +1,196 @@
+import os
```

Comment:
We don't need those tests, this behaviour should be tested end2end in test_huggingface_models